.github/workflows/integration.yml

```diff
@@ -177,6 +177,111 @@ jobs:
+      #       const res = await github.graphql(
+      #         `
+      #         query ($owner: String!, $repo: String!, $pullRequestNumber: Int!) {
+      #           repository(owner: $owner, name: $repo) {
+      #             pullRequest(number: $pullRequestNumber) {
+      #               latestReviews(first: 100) {
+      #                 nodes {
+      #                   author {
+      #                     login
+      #                   }
+      #                   state
+      #                 }
+      #               }
+      #             }
+      #           }
+      #         }
+      #         `,
+      #         {
+      #           owner: context.repo.owner,
+      #           repo: context.repo.repo,
+      #           pullRequestNumber: context.payload.pull_request.number,
+      #         }
+      #       );
+      #       const users = new Set();
+      #       res.repository.pullRequest.latestReviews.nodes.forEach((n) => {
+      #         if(n.state == "APPROVED") {
+      #           users.add(n.author.login)
+      #         }
+      #       });
+      #       const teamRes = await github.graphql(
+      #         `
+      #         query paginate($cursor: String, $owner: String!) {
+      #           organization(login: $owner) {
+      #             teams(first: 100, after: $cursor) {
+      #               totalCount
+      #               nodes {
+      #                 name
+      #                 members(first: 100) {
+      #                   nodes {
+      #                     login
+      #                   }
+      #                 }
+      #               }
+      #               pageInfo {
+      #                 hasNextPage
+      #                 endCursor
+      #               }
+      #             }
+      #           }
+      #         }
+      #         `,
+      #         {
+      #           owner: context.repo.owner,
+      #         }
+      #       );
+      #       const teams = new Set();
+      #       teamRes.organization.teams.nodes.forEach((n) => {
+      #         if(n.members.nodes.some((u) => users.has(u.login))) {
+      #           teams.add(n.name)
+      #         }
+      #       })
+      #       const approverData = {
+      #         "users":  Array.from(users),
+      #         "teams": Array.from(teams)
+      #       };
+      #       const opts = Object.assign({}, { mode: 0o640, flag: 'wx', flush: true }, { encoding: "utf8" });
+      #       await fs.writeFile(`${tmpDir}/approvers.json`, JSON.stringify(approverData), opts);
+
+      - name: 'Setup OPA'
+        uses: 'open-policy-agent/setup-opa@v2'
+        with:
+          version: 'latest'
+
+      - name: 'Evaluate Policy'
+        id: 'opa_eval'
+        env:
+          DIRECTORY: '${{ matrix.working_directory }}'
+          TEMP_DIR: '${{ runner.temp }}'
+        shell: 'bash'
+        run: |-
+          terraform -chdir=${DIRECTORY} show -json tfplan.binary > tfplan.json
```

Comment:
I wonder if we should build this into the plan command to just write the json file too.